layers/auto-completion/README.org

```diff
@@ -105,8 +105,8 @@ To enable docstring tooltips set =auto-completion-enable-help-tooltip= to =t=
 
 #+BEGIN_SRC emacs-lisp
 (setq-default dotspacemacs-configuration-layers
-  '(auto-completion :variables
-                    auto-completion-enable-help-tooltip t))
+  '((auto-completion :variables
+                    auto-completion-enable-help-tooltip t)))
 #+END_SRC
```

Comment:
@robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.